frontend/src/components/ui/question/QuestionBox.js

```diff
@@ -61,6 +76,18 @@ function QuestionBox() {
     }
   }
 
+  // Encodes the roomId into an integer so that the same interview quetion can be
```

Comment:
"Encode" 

Maybe can follow this JS comment guidelines: 
https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide/Code_style_guide/JavaScript#general_guidelines_for_javascript_code_examples

Reply:
Please correct me if I am wrong, this is a wrong way of commenting a function? So rather than placing the comments outside, place it inside like this example? 
![image](https://user-images.githubusercontent.com/56862300/197374721-2b4b8e1b-7880-4c88-b301-338b12f5c090.png)

Reply:
I'm not sure about whether putting outside or inside is the correct way. But in my opinion, putting outside of function scope mean you want to comment on the function as a whole. But if put inside means comment on part of the function implementation. Either way is fine I feel. Correct me if I'm wrong haha